frontend/providers/cloudserver/src/pages/cloudserver/create/index.tsx

```diff
@@ -26,6 +27,11 @@ export default function EditOrder() {
   const { lastRoute } = useGlobalStore();
   const [instanceType, setInstanceType] = useState<CloudServerType>();
 
+  const nanoid = customAlphabet('abcdefghijklmnopqrstuvwxyz', 4);
+  const nanoidUpper = customAlphabet('ABCDEFGHIJKLMNOPQRSTUVWXYZ', 4);
+  const nanoidNumber = customAlphabet('0123456789', 4);
+  const password = nanoid() + nanoidUpper() + nanoidNumber();
```

Comment:
Move the password generation function to the tool file, otherwise the page change will be triggered repeatedly